src/extension.ts

```diff
@@ -31,6 +31,65 @@ export function activate(context: vscode.ExtensionContext) {
 	});
 	context.subscriptions.push(disposable_number_cursors_from_one);
 	
+	let disposable_number_cursors_from_arbitrary = vscode
+			.commands
+			.registerTextEditorCommand('engineernick.multi-cursor-tools.number_cursors_from_arbitrary', (editor, edit, args) => {
+		vscode.window.showInputBox({
+			placeHolder: "1",
+			prompt: "Starting number",
+			value: 1
+		}).then((num)=>{
```

Comment:
I reccomend giving the input argument a different name to distingush between the unparsed/parsed input data;

```suggestion
		}).then(user_input => {
```

Reply:
Makes a lot of sense to me